backend/models/test.model.ts

```diff
@@ -124,8 +125,19 @@ const TestSchema: Schema = new Schema(
       required: true,
     },
     grade: {
-      type: Number,
+      type: String,
       required: true,
+      enum: [
```

Comment:
hmm could you actually make an enum for grade and then refactor all the models to use it (i.e. `enum: Object.keys(Grade),`

Reply:
I could, but then I'd also have to make the objects GRADE_1 example, I was wondering if that'd be a problem for consistency with the other files the use grade as a string? Or should I be changing those files as well?

Reply:
Like the `user.model.ts` file as well?

Reply:
which files are you referring you? if its like "Grade 1", they should convert to `Grade.ONE` or smth

Reply:
yes update user.model.ts and class.model.ts

Reply:
Done!